modules/swagger-codegen/src/main/resources/python/rest.mustache

```diff
@@ -48,66 +20,39 @@ except ImportError:
 logger = logging.getLogger(__name__)
 
 
-class RESTResponse(io.IOBase):
-
-    def __init__(self, resp):
-        self.urllib3_response = resp
-        self.status = resp.status
-        self.reason = resp.reason
-        self.data = resp.data
-
-    def getheaders(self):
-        """
-        Returns a dictionary of the response headers.
-        """
-        return self.urllib3_response.getheaders()
-
-    def getheader(self, name, default=None):
-        """
-        Returns a given response header.
-        """
-        return self.urllib3_response.getheader(name, default)
-
-
-class RESTClientObject(object):
-
-    def __init__(self, pools_size=4):
-        # urllib3.PoolManager will pass all kw parameters to connectionpool
-        # https://github.com/shazow/urllib3/blob/f9409436f83aeb79fbaf090181cd81b784f1b8ce/urllib3/poolmanager.py#L75
-        # https://github.com/shazow/urllib3/blob/f9409436f83aeb79fbaf090181cd81b784f1b8ce/urllib3/connectionpool.py#L680
-        # ca_certs vs cert_file vs key_file
-        # http://stackoverflow.com/a/23957365/2985775
-
-        # cert_reqs
-        if Configuration().verify_ssl:
```

Comment:
There's a requirement from developers to ignore SSL/Cert error when using the Python API client in their development/test environment.